CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ cmake_minimum_required(VERSION 3.3.2)
 
 # Fortran below is needed for SPHEREPACK.
 project(SpECTRE VERSION 0.0.0 LANGUAGES CXX C Fortran)
+message("\n\nPYTHONINTERP_FOUND is ${PYTHONINTERP_FOUND}")
```

Comment:
shouldn't this go in SetupPythonLibs instead of here?

Reply:
I'm guessing this is leftover debugging code.